src/ecs/resources/input/input_handler.rs

```diff
@@ -0,0 +1,329 @@
+//! World resource that handles all user input.
+
+use engine::{ElementState, WindowEvent, Event, VirtualKeyCode, MouseButton};
+use smallvec::SmallVec;
```

Comment:
```
std imports

dependency imports

local imports
```

Could we sort inputs like that?

Also see [the fmt RFC issue](https://github.com/rust-lang-nursery/fmt-rfcs/issues/24)